pkg/util/cgroup/manager/cgroup.go

```diff
@@ -300,3 +302,121 @@ func DropCacheWithTimeoutWithRelativePath(timeoutSecs int, absCgroupPath string,
 
 	return nil
 }
+
+func SetSwapMaxWithAbsolutePathToParentCgroupRecursive(absCgroupPath string) error {
+	if !common.CheckCgroup2UnifiedMode() {
+		general.Infof("[SetSwapMaxWithAbsolutePathToParentCgroupRecursive] is not supported on cgroupv1")
+		return nil
+	}
+	general.Infof("[SetSwapMaxWithAbsolutePathToParentCgroupRecursive] on cgroup: %s", absCgroupPath)
+	swapMaxData := &common.MemoryData{SwapMaxInBytes: math.MaxInt64}
+	err := GetManager().ApplyMemory(absCgroupPath, swapMaxData)
+	if err != nil {
+		return err
+	}
+
+	parentDir := filepath.Dir(absCgroupPath)
```

Comment:
will it access non-cgroup dir and return err?

Reply:
yes, this err if ignored